src/server/saved_objects/saved_objects_mixin.js

```diff
@@ -30,30 +30,34 @@ export function savedObjectsMixin(kbnServer, server) {
     const adminCluster = server.plugins.elasticsearch.getCluster('admin');
 
     try {
-      await adminCluster.callWithInternalUser('cluster.health', {
-        timeout: `${server.config().get('savedObjects.indexCheckTimeout')}ms`,
```

Comment:
__nit:__ Seems like `indexCheckTimeout` is not used anymore, if so could you please remove it completely? I see it only in the config schema (it looks like it's the only property `savedObjects` config node) and in a single test.

Reply:
https://github.com/elastic/kibana/pull/14271/commits/fe35e502a917076bf95d0a9edb1981a81e609535